apps/src/gamelab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -49,48 +50,42 @@ var styles = {
   }
 };
 
-var AnimationPickerListItem = function (props) {
-  var thumbnailStyle = [
-    styles.thumbnail,
-    props.icon && styles.thumbnailIcon
-  ];
+const AnimationPickerListItem = React.createClass({
+  propTypes: {
+    animation: React.PropTypes.shape(METADATA_SHAPE),
+    icon: React.PropTypes.string,
+    label: React.PropTypes.string.isRequired,
+    onClick: React.PropTypes.func
+  },
 
-  var labelStyle = [
-    styles.label,
-    props.icon && styles.labelIcon
-  ];
+  render() {
+    var thumbnailStyle = [
+      styles.thumbnail,
+      this.props.icon && styles.thumbnailIcon
+    ];
 
-  return (
-    <div style={styles.root} onClick={props.onClick}>
-      <div style={thumbnailStyle}>
-        {props.sourceUrl && <AnimationPreview
-            width={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
-            height={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
-            sourceUrl={props.sourceUrl}
-            sourceWidth={props.sourceWidth}
-            sourceHeight={props.sourceHeight}
-            frameWidth={props.frameWidth}
-            frameHeight={props.frameHeight}
-            frameCount={props.frameCount}
-            frameRate={props.frameRate}/>}
-        {props.icon && <i className={"fa fa-" + props.icon} />}
-      </div>
-      <div style={labelStyle}>
-        {props.label}
+    var labelStyle = [
+      styles.label,
+      this.props.icon && styles.labelIcon
+    ];
+
+    return (
+      <div style={styles.root} onClick={this.props.onClick}>
+        <div style={thumbnailStyle}>
+          {this.props.animation &&
+              <AnimationPreview
+                  animation={this.props.animation}
+                  width={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
+                  height={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
+              />
+          }
```

Comment:
I've been not indenting in these cases, so that the indentation effectively represents what the DOM looks like. i.e. I would have done:
```
<div style={thumbnailStyle}>
  {this.props.animation &&
  <AnimationPreview
      animation={this.props.animation}
      width={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
      height={THUMBNAIL_SIZE - 2 * THUMBNAIL_BORDER_WIDTH}
  />
  }
```

Not sure that my way is necessarily better, but wonder if we should call one of these out as the accepted style and then try to be consistent.